validator/sawtooth_validator/journal/chain.py

```diff
@@ -220,6 +222,15 @@ def _verify_block_batches(self, blkw):
                 return False
         return True
 
+    def _validate_permissions(self, blkw):
```

Comment:
Could you add a little doc to this?  What's the context that this is validating permissions for the block.